gpbft/gpbft.go

```diff
@@ -670,7 +674,10 @@ func (i *instance) tryConverge() error {
 		// Else preserve own proposal.
 		fallback, ok := i.getRound(i.round).converged.FindProposalFor(i.proposal)
```

Comment:
Perhaps at this point it would be better to do what the FIP requests, which is to try the next ticket in sequence rather than wait to receive our own. It still might not be a viable candidate though, so we can still end up here.